protobuf/drand/control.proto

```diff
@@ -89,6 +89,8 @@ message InitDKGPacket {
     uint32 beacon_period = 3;
     // the minimum beacon period when in catchup.
     uint32 catchup_period = 4;
+    // indicates if the node should not use the prev sig to generate the next one
+    bool decouple_prev_sig = 5;

```

Comment:
I would call this "Version" really - like that the code knows what it should do with any parameters found and we can extend this message with other parameters depending on the version, like `V2Params` and `V3Params` union for example.